query/call_iterator.go

```diff
@@ -1247,6 +1247,32 @@ func newMovingAverageIterator(input Iterator, n int, opt IteratorOptions) (Itera
 	}
 }
 
+// newExponentialMovingAverageIterator returns an iterator for operating on an exponential_moving_average() call.
+func newExponentialMovingAverageIterator(input Iterator, n int, nHold int, warmupType string, opt IteratorOptions) (Iterator, error) {
```

Comment:
`n, nHold int` you don't need to repeat the argument twice.

The argument for `warmupType` should be the `gota.WarmupType`. The string to warmup type can be somewhere else.